core/src/test/java/nl/nn/adapterframework/jdbc/FixedQuerySenderTest.java

```diff
@@ -66,213 +93,214 @@ private void assertSenderException(SenderException ex) {
-		sender.setQuery("INSERT INTO "+JdbcTestBase.TEST_TABLE+" (tKEY, tVARCHAR) VALUES ('1', ?{param})");
-		sender.setUseNamedParams(false);
-		sender.configure();
-		sender.open();
+	@DatabaseTest
+	public void testNamedParamInQueryFlagFalse(DatabaseTestEnvironment databaseTestEnvironment) throws Exception {
+		fixedQuerySender.setQuery("INSERT INTO " + TABLE_NAME + " (tKEY, tVARCHAR) VALUES ('1', ?{param})");
+		fixedQuerySender.setUseNamedParams(false);
+		fixedQuerySender.configure();
+		fixedQuerySender.open();
 
-		SenderException ex = assertThrows(SenderException.class, () -> sendMessage("dummy"));
+		SenderException ex = assertThrows(SenderException.class, () -> fixedQuerySender.sendMessage(new Message("dummy"), session));
 
-		assertSenderException(ex);
+		assertSenderException(databaseTestEnvironment.getDataSourceName(), ex);
 	}
 
-	@Test
-	public void testIncompleteNamedParamInQuery() throws Exception {
-		sender.setQuery("INSERT INTO "+JdbcTestBase.TEST_TABLE+" (tKEY, tVARCHAR) VALUES ('1', ?{param)");
-		sender.configure();
-		sender.open();
+	@DatabaseTest
+	public void testIncompleteNamedParamInQuery(DatabaseTestEnvironment databaseTestEnvironment) throws Exception {
+		fixedQuerySender.setQuery("INSERT INTO " + TABLE_NAME + " (tKEY, tVARCHAR) VALUES ('1', ?{param)");
+		fixedQuerySender.configure();
+		fixedQuerySender.open();
 
-		SenderException ex = assertThrows(SenderException.class, () -> sendMessage("dummy") );
+		SenderException ex = assertThrows(SenderException.class, () -> fixedQuerySender.sendMessage(new Message("dummy"), session));
 
-		assertSenderException(ex);
+		assertSenderException(databaseTestEnvironment.getDataSourceName(), ex);
 	}
 
-	@Test
+	@DatabaseTest
 	public void testMultipleColumnsReturnedWithSpaceBetween() throws Exception {
-		assumeThat(productKey, anyOf(is("H2"),is("Oracle")));
-		sender.setQuery("INSERT INTO "+JdbcTestBase.TEST_TABLE+" (tKEY, tVARCHAR) VALUES ('1', ?)");
-		sender.addParameter(new Parameter("param1", "value"));
+		assertThat(dataSourceName, anyOf(is("H2"), is("Oracle")));
+		fixedQuerySender.setQuery("INSERT INTO " + TABLE_NAME + " (tKEY, tVARCHAR) VALUES ('1', ?)");
+		fixedQuerySender.addParameter(new Parameter("param1", "value"));
 
-		sender.setColumnsReturned("tKEY, tVARCHAR");
+		fixedQuerySender.setColumnsReturned("tKEY, tVARCHAR");
 
-		sender.configure();
-		sender.open();
+		fixedQuerySender.configure();
+		fixedQuerySender.open();
 
-		Message result=sendMessage("dummy");
+		Message result = fixedQuerySender.sendMessage(new Message("dummy"), session).getResult();
 		assertColumnsReturned(result);
 	}
 
-	@Test
+	@DatabaseTest
 	public void testMultipleColumnsReturnedWithDoubleSpace() throws Exception {
-		assumeThat(productKey, anyOf(is("H2"),is("Oracle")));
-		sender.setQuery("INSERT INTO "+JdbcTestBase.TEST_TABLE+" (tKEY, tVARCHAR) VALUES ('1', ?)");
-		sender.addParameter(new Parameter("param1", "value"));
+		assertThat(dataSourceName, anyOf(is("H2"), is("Oracle")));
```

Comment:
En hier ook

Reply:
Dat geldt ook voor hier.
tenzij het voor kan komen dat we helemaal niet met H2 testen...